app/src/main/res/layout-land/promoted_story_list.xml

```diff
@@ -49,13 +50,21 @@
         android:textSize="14sp" />
     </LinearLayout>
 
-    <androidx.recyclerview.widget.RecyclerView
+    <org.oppia.android.app.home.topiclist.PromotedStoryListView
       android:id="@+id/promoted_story_list_recycler_view"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:clipToPadding="false"
       android:layout_marginTop="4dp"
       android:overScrollMode="never"
-      android:scrollbars="none" />
+      android:scrollbars="none"
+      android:orientation="horizontal"
+      android:paddingStart="@{viewModel.paddingStart}"
```

Comment:
Ah, per my earlier comment maybe we need both a paddingStart & paddingEnd (where the latter is computed). I'm not actually sure--this is confusing me a bit as a reader.

Reply:
Ah I think I get what you mean now -- I'll change these to getXX functions for clarity, even though paddingStart does not change and hopefully that will be clearer.